docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "2"
+services:
+  app:
+    build: .
+    environment:
+      - NODE_ENV=docker-compose
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+    links:
+      - redis
+    ports:
+      - "8080:8080"
+    volumes:
+      - "/public"
+  redis:
+    image: redis
+
+  nginx-proxy:
+    image: quay.io/ukhomeofficedigital/nginx-proxy:v1.2.3
```

Comment:
the latest nginx-proxy is 1.5.1 I'd suggest updating to that as it's a fresh project